packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -38,27 +40,114 @@ const GlobalClassNames = {
   custom: 'ms-Panel--custom'
 };
 
-const panelSize = {
-  width: {
-    xs: '272px',
-    sm: '340px',
-    md: '643px',
-    lg: '940px'
+const panelWidth = {
+  full: '100%',
+  auto: 'auto',
+  xs: 272,
+  sm: 340,
+  md1: 592,
+  md2: 644,
+  lg: 940
+};
+
+const panelMargin = {
+  auto: 'auto',
+  none: 0,
+  md: 48,
+  lg: 428,
+  xl: 176
+};
+
+const smallPanelSelectors = {
+  ['@media (min-width: ' + ScreenWidthMinMedium + 'px)']: {
```

Comment:
Nitpicky thing to keep in mind (you don't necessarily need to change it here), consider using the new ` `` ` style strings if you have a variable in the middle of the string. Bit easier to read (and write!) and more compact.
```
`@media (min-width: ${ScreenWidthMinMedium}px)`
```
(in related news, figuring out how to write literal backticks in markdown is interesting...!)

Reply:
Just copied over and overlooked. Fixing 👍